packages/picongpu/picongpu.profile

```diff
@@ -7,3 +7,11 @@ spack load @PIC_SPACK_COMPILER@ >/dev/null 2>&1 || echo "using default compiler"
 
 # required: PIConGPU and its dependencies
 spack load -r @PIC_SPACK_SPEC@
+
+# activate bash completion if available
+BASH_COMP_FILE=$PICSRC/bin/picongpu-completion.bash
+if [ -f $BASH_COMP_FILE ] ; then
+    source $PICSRC/bin/picongpu-completion.bash
```

Comment:
looks like I slept during adding this to PIConGPU. Could you please change this line to `source $BASH_COMP_FILE`

Reply:
oops, and I didn't see it when I was copying it :D